src/test/java/uk/gov/hmcts/reform/professionalapi/controller/advice/ExceptionMapperTest.java

```diff
@@ -80,7 +76,7 @@ public void should_handle_http_media_type_not_supported_exception() {
 
     @Test
     public void should_handle_forbidden_error_exception() {
-        AccessDeniedException exception = mock(AccessDeniedException.class);
+        AccessDeniedException exception = new AccessDeniedException("Access Denied");
 
```

Comment:
if possible verify the error message otherwise not creating any value add of this test?

Reply:
This has been addressed in this commit: https://github.com/hmcts/rd-professional-api/pull/450/commits/2e6a2f8b1a7050bb52d904bd20a2c5638869a566